pagopa/io-app-design-system
# feat: [SIW-470] Add PSP as navigation icon

## Short description
This PR adds the `psp` icon as a `IONavIcons` as shown in the IT-Wallet FIGMA. 

## List of changes proposed in this pull request
- Adds a new `navPsp` property in the `IOIcons` and `IONavIcons` objects.

## How to test
Run the example app and check if the icon gets rendered in the `Icons -> Navigation` section.
